### PR TITLE
Add ut for service.go

### PR DESCRIPTION
**Pull request by jwcesign**
Signed-off-by: jwcesign <jiangwei115@huawei.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Add ut for service.go

**Which issue(s) this PR fixes**:
Part of https://github.com/karmada-io/karmada/issues/2579

**Special notes for your reviewer**:
none

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by jwcesign** [comment]
cc @XiShanYongYe-Chang

**Comment by jwcesign** [comment]
cc @XiShanYongYe-Chang  for checking

File: pkg/karmadactl/cmdinit/kubernetes/services_test.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+			svc: []*corev1.Service{
+				{
+					ObjectMeta: metav1.ObjectMeta{Name: "foo"},
+					Spec:       corev1.ServiceSpec{Type: corev1.ServiceTypeClusterIP, Ports: []corev1.ServicePort{{Port: 30000}}},
+				},
+			},
+			exists: false,
+		},
+		{
+			name: "there is node port svc",
```
Comment (XiShanYongYe-Chang) [inline pkg/karmadactl/cmdinit/kubernetes/services_test.go:82]: How about adding one more case: `exist a NodePort type service with the same nodePort`

**Review by XiShanYongYe-Chang — COMMENTED** [review]
/lgtm

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**